TTS/config/shared_configs.py

```diff
@@ -251,11 +251,27 @@ class BaseTrainingConfig(TrainerConfig):
 
         num_eval_loader_workers (int):
             Number of workers for evaluation time dataloader.
+
+        use_speaker_weighted_sampler (bool):
+            Enable / Disable the batch balancer by speaker. Defaults to ```False```.
+
+        speaker_weighted_sampler_alpha (float):
+            Number that control the influence of the speaker sampler weights. Defaults to ```1.0```.
+
+        use_language_weighted_sampler (bool):
+            Enable / Disable the batch balancer by language. Defaults to ```False```.
+
+        language_weighted_sampler_alpha (float):
+            Number that control the influence of the language sampler weights. Defaults to ```1.0```.
```

Comment:
These arguments are supposed to be used only with TTS models right?

If so why don't you move these to shared TTS configs?

Reply:
Indeed, when I implemented it I was planning to use it in the emotion encoder. But it does not make more sense after the implementation of the new sampler.

Reply:
Done